pages/LearnerDashboard.html

```diff
@@ -36,11 +36,11 @@ <h2>
     The Subscriptions section:
   </p>
 
-  <img src="static/images/subscriptionsLearnerDashboard.png" style="width: 100%;">
+  <img src="static/images/subscriptionsLearnerDashboard.png" style="width: 100%;" alt="The left-hand navigation bar has 'Subscriptions' highlighted. Three subscriptions are displayed. The 'sort-by' dropdown list reads 'username.'">
 
   <p>
   You can also select the "Sort By" dropdown to sort your explorations and collections by last played, title, and category.
   </p>
 
-  <img src="static/images/sortingOption.png" style="width: 100%;">
+  <img src="static/images/sortingOption.png" style="width: 100%;" alt="The navigation bar on the left has 'In-progress explorations' highlighted. The 'sort-by' dropdown list reads 'Last-Played' and is circled in yellow.">
```

Comment:
Prefix this and the previous description with something like "A screenshot of the learner dashboard.", to set context. Check that all your other descriptions set context.